.changeset/moody-items-wait.md

```
@@ -0,0 +1,12 @@
+---
+'@mastra/client-js': patch
+'@mastra/server': patch
+'@mastra/mongodb': patch
+'@mastra/spanner': patch
+'@mastra/core': patch
+'@mastra/libsql': patch
+'@mastra/mysql': patch
+'@mastra/pg': patch
+---
+
+Round-trip dataset item metadata in experiment results.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)